src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/RazorFormattingPassBase.cs

```diff
@@ -126,7 +128,7 @@ directiveCode.Children is [RazorDirectiveSyntax directive] &&
 
         return false;
 
-        static bool TryGetWhitespace(SyntaxList<RazorSyntaxNode> children, [NotNullWhen(true)] out CSharpStatementLiteralSyntax? whitespaceBeforeSectionName, [NotNullWhen(true)] out UnclassifiedTextLiteralSyntax? whitespaceAfterSectionName)
+        static bool TryGetWhitespace(AspNetCore.Razor.Language.Syntax.SyntaxList<RazorSyntaxNode> children, [NotNullWhen(true)] out CSharpStatementLiteralSyntax? whitespaceBeforeSectionName, [NotNullWhen(true)] out UnclassifiedTextLiteralSyntax? whitespaceAfterSectionName)
```

Comment:
Consider adding a using type alias at the type of the file for SyntaxList<RazorSyntaxNode>:

```C#
using RazorSyntaxNodeList = Microsoft.AspNetCore.Razor.Language.Syntax.SyntaxList<Microsoft.AspNetCore.Razor.Language.Syntax.RazorSyntaxNode>;
```

Reply:
@davidwengier: It feels like some global usings for common type aliases like these might be useful. What do you think?

Reply:
> @davidwengier: It feels like some global usings for common type aliases like these might be useful. What do you think?

I personally would love that for consistency, totally open to suggestions here.

Reply:
Also since there is a list of SyntaxNode and a list of RazorSyntaxNode, I ended up with RazorSyntaxNodeList and RazorRazorSyntaxNodeList :) The latter sound silly though follows the pattern. Suggestions are welcome.

Reply:
I honestly do not understand why Razor has both `SyntaxNode` and `RazorSyntaxNode`. It's very unclear and inconsistent. `RazorSyntaxNode` inherits directly from `SyntaxNode` and adds no extra implementation. _Most_ `SyntaxNode` subtypes in the Razor compiler actually inherit from `RazorSyntaxNode`. Heck, even `CSharpSyntaxNode` inherits from `RazorSyntaxNode`. There are only a few types that directly inherit from `SyntaxNode`, and I wonder if those are mistakes? For example, `SyntaxToken` inherits from `RazorSyntaxNode` but `SyntaxTrivia` inherits from `SyntaxNode`? It's all wacky. :smile:

Reply:
Maybe @333fred, @chsienki, or @jjonescz understand some nuance between `SyntaxNode` and `RazorSyntaxNode` that I don't see?

Reply:
I certainly don't know of any nuances between these. While I suspect there may have once been an attempt to separate razor from C#/VB, as you mentioned it doesn't exist today. The whole syntax implementation is a bit wacky, given the whole "copied from pre-CTP 1 Roslyn" nature of it.

Reply:
Yeah, I just chatted with @chsienki and he didn't know of any reason. It doesn't look to me like anything actually creates `SyntaxTrivia` instances. That one could probably just be removed.

Reply:
We're thinking about using `SyntaxTrivia` if the request for razor preprocessor directives ever goes forward.

Reply:
> @davidwengier: It feels like some global usings for common type aliases like these might be useful. What do you think?

No complaints. These type names have always been annoying. Sometimes `SyntaxNode` is a razor node, sometimes its a roslyn node 🤷‍♂️